stock_move_backdating/models/stock_inventory.py

```diff
@@ -21,7 +21,7 @@ def onchange_date_backdating(self):
         check_date(self.date_backdating)
 
     def post_inventory(self):
-        no_backdate_inventories = self.env["stock.inventory"].browse()
+        no_backdate_inventories = self.env["stock.inventory"]
```

Comment:
@toita86 The former writing was the good one.

Reply:
Why is it? We get the same result?

Reply:
Yes, but formally this is better as instance is not really the same as a browse record set. Nevertheless, this one is not blocking